Day2ConsoleAppHomework/Program.cs

```diff
@@ -1,15 +1,44 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Day2ConsoleAppHomework
 {
+    enum TransactionType { Buy, Sell }
+
     class Program
     {
+        string currency = "DKK";
+
         static void Main(string[] args)
         {
+            string currency = "DKK";
+
+            Console.WriteLine("Input the transaction type:");
+            var userInput = Console.ReadLine();
+            var transactionType = (TransactionType)Enum.Parse(typeof(TransactionType), userInput, true);
+
+            Console.WriteLine("Input the nominal of the trade:");
+            userInput = Console.ReadLine();
+            var nominal = float.Parse(userInput);
```

Comment:
var nominal = float.Parse(Console.ReadLine());

Reply:
I think I understand what you mean. I made a new attempt.
 var nominal = float.Parse(Console.ReadLine());